test/models/battleModel.test.js

```diff
@@ -0,0 +1,28 @@
+const {
+  expect, dropDatabase, battleModel, assert, models,
+} = require('test/testHelper');
+const { BattleFactory } = require('test/factories');
+
+describe('battleModel', async () => {
+  describe('createNewBattle', async () => {
+    let battleData;
+
+    beforeEach(async () => {
+      await dropDatabase();
+      battleData = await BattleFactory.createNewBattle({ cryptoName: 'BTC' });
+    });
+    it('should save new battle to db', async () => {
+      const actual = await battleModel.createNewBattle(battleData);
+      assert(!actual.isNew); // if actual is saved to db it is not new
+    });
+    it('should find battle', async () => {
+      await battleModel.createNewBattle(battleData);
+      const battle = await models.Battle.findOne({ 'playersInfo.firstPlayer.cryptoName': 'BTC' });
+      expect(battle.playersInfo.firstPlayer.cryptoName).to.be.eq('BTC');
+    });
+    it('should be error message', async () => {
+      const { message } = await battleModel.createNewBattle({ lkkdfs: 'dfsdf' });
+      expect(message).to.be.exist;
+    });
```

Comment:
I don’t quite understand what you are doing if you want to test the creation of a battle. You don’t need to create it in the factory, look at the parser how we use the factory to return only data, without creating a record, and do the same

Reply:
done